### DISCUSSION TITLE
Use `docker logs -f nginx-proxy` looks so weird. How can I use Goaccess to parse nginx-proxy log?

### DISCUSSION THREAD

**Discussion by nguyenleminhquan** [nginx-proxy/nginx-proxy#2368] [Q&A] [answered]
Hi everyone,

I deploy nginx-proxy and then use `docker logs -f nginx-proxy` to see its log, so one line of log look like:
`nginx.1     | 34.29.69.150 45.13.119.147 - - [05/Jan/2024:03:00:06 +0000] "POST /boaform/admin/formLogin HTTP/1.1" 503 190 "http://34.29.69.150:80/admin/login.asp" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:71.0) Gecko/20100101 Firefox/71.0" "-"`
It looks quite weired that `nginx.1     |` is the reason that i can't use Goaccess to parse log. 

How can I remove `nginx.1     |` and make log look like:
`34.29.69.150 45.13.119.147 - - [05/Jan/2024:03:00:06 +0000] "POST /boaform/admin/formLogin HTTP/1.1" 503 190 "http://34.29.69.150:80/admin/login.asp" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:71.0) Gecko/20100101 Firefox/71.0" "-"`

I'm not good at English grammar so wish you sympathize.
Thank you for reading my question!

**Comment by buchdag** [comment, marked as answer]
Hi.

The `nginx.1     |` comes from the fact that the nginx-proxy container runs two processes (nginx and [docker-gen](https://github.com/nginx-proxy/docker-gen)), handled by a process manager ([forego](https://github.com/nginx-proxy/forego)). The process manager label the log output so you can tell which process outputted each given log line.

You can run nginx and docker-gen each in its own container by following [those instructions](https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#separate-containers). The separate containers won't have the `process.x   |` added by forego.

**Reply by nguyenleminhquan** [reply]
Thank you. I will try to do it the way you suggested. I will tell the results later.

**Reply by nguyenleminhquan** [reply]
Hi Nicolas,

Can I stream the nginx-proxy access log to a file? There appears to be no way to make this :(.

Thank you for reading my question.
